File_Engine/Convert/ToJsonArray.cs

```diff
@@ -0,0 +1,84 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.Engine.Serialiser;
+using BH.oM.Adapter;
+using BH.oM.Adapters.File;
+using BH.oM.Base;
+using BH.oM.Base.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.Engine.Adapters.File
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /*** Methods                                     ***/
+        /***************************************************/
+
+        [Description("Serialises the objects and returns a json array.")]
+        [Input("objects", "Objects you want to serialize and concatenate into a single JSON array.")]
+        [Output("jsonArray", "A JSON array with the serialized objects.")]
+        public static string ToJsonArray(this List<object> objects)
```

Comment:
What does this do differently from `BH.Engine.Serialiser.Convert.ToJsonArray`? Or is this also a left over of previous things?

Reply:
Good point. Kinda leftover from previous change where I forgot that the base Serialiser_Engine is able to treat Json Arrays now. However, this in addition allows to wrap non-bhomObjects and write primitive types, see example script, which the base serialiser can not do, and was requested by users.
Could be made a feature of the base Serialiser method. Could we agree on making this `private` in the meantime and keep it?

Reply:
I'm referring to this 
<img width="849" alt="image" src="https://user-images.githubusercontent.com/6352844/149920891-bf033e47-9538-4894-9b98-176228ab3e6c.png">

Reply:
Ok that makes sense, happy with this being `private` for now then and move a discussion on implementation into the Serialiser Engine for long  term 👍